MiniApp/Classes/core/Extensions/NotificationCenter+RAT.swift

```diff
@@ -3,21 +3,23 @@ internal extension NotificationCenter {
         sendAnalytics(event: event, type: type, parameters: customData)
     }
 
-    func sendAnalytics(event: MiniAppRATEvent, type: MiniAppRATEventType? = nil, parameters customData: [(String, String)]? = nil) {
+    func sendAnalytics(event: MiniAppRATEvent, type: MiniAppRATEventType? = nil, parameters customData: [(String, String)]? = nil, analyticsConfig: [MAAnalyticsConfig] = []) {
```

Comment:
see all my other comments about `analyticsConfig` needing to contain `MAAnalyticsConfig(acc: "1553", aid: "1")` even if it is not passed :)

Reply:
Done